eslint.config.mjs

```diff
@@ -0,0 +1,9 @@
+import unjs from "eslint-config-unjs";
+
+export default unjs({
+  ignores: ["dist", "node_modules", "src/_parser.ts", "coverage"],
```

Comment:
I'm pretty sure `dist` and `node_modules` are already ignored (otherwise, we should ignore them in the shared config).

`coverage` should be ignore in the shared config.

Why ignoring `src/_parser.ts`.

Reply:
Hello @Barbapapazes , thank you for your comment.

I checked the settings for `dist` and coverage in `eslint-config-unjs`. Therefore, I will delete it.

- https://github.com/unjs/eslint-config/blob/3b3ff89f1f8502f229fc776bbf24bacf9bdf9433/src/eslint.config.ts#L71

However, with the current settings, `node_modules` was not included in this (I don't know why).

`src/_parser.ts` was defined in the deleted `.eslintignore`.
I don't know why I added it, so I left it as is.